### PR TITLE
Validation - Individual Census (England & Wales)

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
[Trello](https://trello.com/c/TGGnRw4v/2940-validation-individual-census-england-wales-s)

Validation added to the Individual Census.

### How to review 
- Ensure tasks on Tello has been complete.
- Does the schema still work as expected?

### CODE REVIEW

File: scripts/test_schemas.sh [hunk 1]
```diff
@@ -6,8 +6,8 @@ if [ "$1" == "--local" ] || [ "$2" == "--local" ]; then
 fi
 
 if [ "$run_docker" == true ]; then
-    docker pull onsdigital/eq-schema-validator:v3
-    validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator:v3)"
+    docker pull onsdigital/eq-schema-validator:allow-default-answer-for-radio
+    validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator:allow-default-answer-for-radio)"
```
Comment (MebinAbraham) [inline scripts/test_schemas.sh]: Reminder to revert before merging.

File: data-source/jsonnet/common/blocks/individual/personal-details/name.jsonnet [hunk 2]
```diff
@@ -25,8 +25,13 @@ local question(title) = {
     {
       id: 'last-name',
       label: 'Last name',
-      mandatory: false,
+      mandatory: true,
       type: 'TextField',
+      validation: {
+        messages: {
+          MANDATORY_TEXTFIELD: 'Please enter a last name to continue',
```
Comment (ajmaddaford) [inline data-source/jsonnet/common/blocks/individual/personal-details/name.jsonnet]: The 'Please' needs to dropped from these error messages (checked with @charlotte-hirst and Donna).

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: data-source/jsonnet/common/blocks/individual/personal-details/name.jsonnet (hunk 2 above)
Comment (bitdivision) [reply data-source/jsonnet/common/blocks/individual/personal-details/name.jsonnet]: Removed from messages in that file. I assume all the labels elsewhere are still okay to use `Please`?

Comment (ajmaddaford) [reply data-source/jsonnet/common/blocks/individual/personal-details/name.jsonnet]: Good spot, probably not. I'd say it shouldn't block this going in, but should be raised with @charlotte-hirst.

**Review by ajmaddaford — APPROVED** [review]

**Review by iwootten — APPROVED** [review]

**Pull request merged**